cli/pcluster/config/mappings.py

```diff
@@ -170,21 +185,28 @@
 }
 
 SCALING = {
-    "type": Section,
+    "type": CfnSection,
     "key": "scaling",
     "default_label": "default",
     "params": {
         "scaledown_idletime": {
-            "type": IntParam,
+            "type": IntCfnParam,
             "default": 10,
             "cfn_param_mapping": "ScaleDownIdleTime",
             "update_policy": UpdatePolicy.COMPUTE_FLEET_STOP
-        }
+        },
+        "_scaledown_idletime": {
+            "type": ScaleDownIdleTimeJsonParam,
+            "default": 10,
+            # This param is managed automatically (it's a copy of scaledown_idletime for Json)
+            "update_policy": UpdatePolicy.IGNORED,
```

Comment:
Uhm I'm not sure. Maybe currently this would be valid but it's not necessarily true that a parameter with ``PRIVATE`` visibility should be ignored if updated. I'd prefer keeping things separated for now, although I recognize that yours is an interesting observation.

Reply:
ok